src/content/docs/how-to/local.mdx

```diff
@@ -1,12 +1,171 @@
 ---
 title: How to build and test locally
-description: A guide in my new Starlight docs site.
+description: This guide shows you how to use the Bluebuild CLI tool to test images in your local development environment before pushing them to GitHub.
 ---
 
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+[bluebuild-cli-repo]: https://github.com/blue-build/cli
+[bluebuild-cli-readme]: https://github.com/blue-build/cli?tab=readme-ov-file
+[bluebuild-cli-readme-howtouse]: https://github.com/blue-build/cli?tab=readme-ov-file#how-to-use
+[bluebuild-cli-ghaction]: https://github.com/blue-build/github-action/
+[bluebuild-cli-ghaction-build]: https://github.com/blue-build/github-action/blob/main/action.yml#L89-L99
+[sample-bug-report]: https://github.com/blue-build/cli/issues/145
+
+[refactor-discussion]: https://github.com/blue-build/cli/issues/65
+
+If you want to do any of the following things locally, without depending on the [BlueBuild GitHub Action][bluebuild-cli-ghaction]:
+
+- preview the `Containerfile` generated your `recipe.yml` locally;
+- test your `recipe.yml` build process locally before pushing to GitHub;
+- install a locally built image onto a virtual machine;
+- rebase your currently running operating system onto a locally built image;
+
+This guide shows you how to use [Bluebuild CLI][bluebuild-cli-repo] to do so.
+
+## Installing BlueBuild CLI
+
+There are several methods available for installing the BlueBuild CLI tool, documented in the the [BlueBuild CLI README][bluebuild-cli-readme].
+
+## Using BlueBuild CLI
+
+The [repository README][bluebuild-cli-readme-howtouse] has general instructions for using the `bluebuild` CLI tool.
+
+There is also a `help` command:
+
+```bash
+$ bluebuild help
+A CLI tool built for creating Containerfile templates based on the Ublue Community Project
+
+Usage: bluebuild [OPTIONS] <COMMAND>
+
+Commands:
+  build        Build an image from a recipe
+  template     Generate a Containerfile from a recipe
+  upgrade      Upgrade your current OS with the local image saved at `/etc/bluebuild/` [aliases: update]
+  rebase       Rebase your current OS onto the image being built
+  bug-report   Create a pre-populated GitHub issue with information about your configuration
+  completions  Generate shell completions for your shell to stdout
+  help         Print this message or the help of the given subcommand(s)
+
+Options:
+  -v, --verbose...  Increase logging verbosity
+  -q, --quiet...    Decrease logging verbosity
+  -h, --help        Print help
+  -V, --version     Print version
+```
```

Comment:
I'm not sure about statically including the whole output of the help command on here, as that is subject to updates, and is available for all local users.

Reply:
Agree